config/locales/time_formats.yml

```diff
@@ -0,0 +1,15 @@
+en:
```

Comment:
Just a clarification. This change isn't going to affect the existing data export is it?

Reply:
Nope it shouldn't as the path to the translation hasn't changed